sdk/js/src/service/devices/index.js

```diff
@@ -183,12 +188,13 @@ class Devices {
 
   async create (applicationId, device, { abp = false, setDefaults = true, withRootKeys = false } = {}) {
     let dev = device
+    const Url = URL ? URL : window.URL
```

Comment:
```suggestion
    const Url = window.URL ? window.URL : URL
```

Reply:
This would error in NodeJS. Why do you want it changed?

Reply:
Will merge now, pls open an issue if I missed something here.